toolkit/tools/imager/imager.go

```diff
@@ -590,6 +591,15 @@ func buildImage(mountPointMap, mountPointToFsTypeMap, mountPointToMountArgsMap,
 	}
 	timestamp.StopEvent(nil) // install chroot packages
 
+	// Configure rpm install macros for the setup environment. We run 'rpm' from outside the install chroot since it starts
+	// empty. So the macros must be defined here before we install packages.
```

Comment:
Does that mean the resulted image won't have the macros?

Reply:
I call this twice, so both the setup environment and the final image should have them. I initially had a second set of flags for the final image but I think this is simpler.